src/settings-ui/Settings.UI/OOBE/Views/OobePowerOCR.xaml

```diff
@@ -0,0 +1,40 @@
+﻿<Page
+    x:Class="Microsoft.PowerToys.Settings.UI.OOBE.Views.OobePowerOCR"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Microsoft.PowerToys.Settings.UI.OOBE.Views"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    xmlns:controls="using:Microsoft.PowerToys.Settings.UI.Controls"
+    xmlns:toolkitcontrols="using:CommunityToolkit.WinUI.UI.Controls">
+
+    <controls:OOBEPageControl x:Uid="Oobe_PowerOCR"
+                              HeroImage="ms-appx:///Assets/Modules/OOBE/PowerOCR.gif">
+
+        <controls:OOBEPageControl.PageContent>
+            <StackPanel Orientation="Vertical">
+                <TextBlock x:Uid="Oobe_HowToUse"
+                           Style="{ThemeResource OobeSubtitleStyle}" />
+
+                <controls:ShortcutWithTextLabelControl x:Name="HotkeyControl" x:Uid="Oobe_PowerOCR_HowToUse" />
+              
+
+                <TextBlock x:Uid="Oobe_TipsAndTricks"
+                           Style="{ThemeResource OobeSubtitleStyle}"/>
+
+                <toolkitcontrols:MarkdownTextBlock Background="Transparent" x:Uid="Oobe_PowerOCR_TipsAndTricks" />
+               
+                <StackPanel Orientation="Horizontal" Spacing="12" Margin="0,24,0,0">
+                    <Button x:Uid="Launch_PowerOCR" Style="{StaticResource AccentButtonStyle}" Click="Start_PowerOCR_Click"/>
```

Comment:
Can be remove as it opens only the settings page. Two buttons with the same action makes no sens.